tests/framework/installer/install_data.go

```diff
@@ -222,11 +222,11 @@ metadata:
   name: rook-tools
   namespace: ` + namespace + `
 spec:
+  dnsPolicy: ClusterFirstWithHostNet
```

Comment:
in the tests you could also remove a sleep since the toolbox doesn't have to wait for the api service to initialize anymore. https://github.com/jbw976/rook/blob/87a9b622c3e8d89374147b6c63f368769e3997a2/tests/framework/installer/install_helper.go#L218

Reply:
ah nice, that would be good to remove that sleep.